cargo-nextest/src/dispatch.rs

```diff
@@ -696,6 +696,16 @@ struct TestReporterOpts {
         env = "NEXTEST_FINAL_STATUS_LEVEL"
     )]
     final_status_level: Option<FinalStatusLevelOpt>,
+
+    /// Visibility of the progress bar.
+    #[clap(
+        long,
+        possible_values = &["true", "false"],
+        conflicts_with_all = &["no-capture", "no-run"],
```

Comment:
I'm not 100% sure about these.

Reply:
Yeah, I think we shouldn't conflict on these.